projects/plugins/jetpack/extensions/blocks/subscriptions/view.js

```diff
@@ -55,6 +91,13 @@ domReady( function () {
 			// Injects loading animation in hidden state
 			button.insertAdjacentHTML( 'beforeend', spinner );
 
+			const emailInput = form.querySelector( 'input[type=email]' );
```

Comment:
You could move the `emailInput` inside `handleIframeMessage` and loop through multiple inputs like we do here with forms:

https://github.com/Automattic/jetpack/blob/270f8b982c8424978a3b6106bb90bdd41db09cf5/projects/plugins/jetpack/extensions/blocks/subscriptions/view.js#L84-L85

Reply:
Ended up with a different approach in https://github.com/Automattic/jetpack/pull/39633/commits/f2c79337d3c6073647aa39737444ef0634385d97.